.travis.yml

```diff
@@ -11,4 +12,4 @@ install:
   - "pip install -r requirements.txt"
   - "pip install ."
 script:
-  - py.test
+  - tox
```

Comment:
So should I then add `requests>=2.9.1` instead while removing the other versions?